exporters/vanillas/kube-state-metrics.yaml

```diff
@@ -0,0 +1,336 @@
+---
```

Comment:
As mentioned earlier in the Makefile, lets move these to the folder with kustomization.yaml and if we add the version to the output file, Make will know when they need updating and when they don't.